extensions/mini/__init__.py

```diff
@@ -71,3 +81,19 @@ def register(self):
         parser = argparse.ArgumentParser(description=self.bot.processOutput("Ping the bot", message))
         args = parser.parse_args(args)
         await message.channel.send("{:0.0f}ms".format(self.bot.latency * 1000))
+
+    async def color(self, args, message):
+        parser = argparse.ArgumentParser(description=self.bot.processOutput("Obtain a colo(u)r from a role on the server", message))
+        parser.add_argument("role", help=self.bot.processOutput("The role to get the color of", message))

```

Comment:
In the future maybe have it default to the author's primary role, but that isn't needed right now.

Reply:
????

Reply:
So if you ran `?color` without giving a role it would display helper since that is your primary role on the server.